alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/util/ConfigurationFieldModelConverter.java

```diff
@@ -122,7 +127,10 @@ public FieldModel convertToFieldModel(final ConfigurationModel configurationMode
     }
 
     public ConfigurationModel convertToConfigurationModel(FieldModel fieldModel) throws AlertDatabaseConstraintException {
-        long descriptorId = descriptorAccessor.getRegisteredDescriptorByName(fieldModel.getDescriptorName()).map(RegisteredDescriptorModel::getId).orElse(0L);
+        String descriptorName = fieldModel.getDescriptorName();
+        DescriptorKey descriptorKey = descriptorMap.getDescriptorKey(descriptorName).orElseThrow(() -> new AlertDatabaseConstraintException("Could not find a Descriptor with the name: " + descriptorName));
+
+        long descriptorId = descriptorAccessor.getRegisteredDescriptorByName(descriptorKey.getUniversalKey()).map(RegisteredDescriptorModel::getId).orElse(0L);
```

Comment:
I'm seeing us get a descriporName string, convert it to a key, then convert it back to a string in many places. We should instead make things like DescriptorAccessor require the use of a DescriptorKey.